src/components/Header/index.tsx

```diff
@@ -125,7 +126,7 @@ export default function Header() {
           className=" grow cursor-pointer py-2 transition-colors hover:bg-sciquelHover"
           style={{ display: scroll ? "none" : "block" }}
         >
-          <Link href="/">LATEST</Link>
+          <Link href="/stories/latest">LATEST</Link>
```

Comment:
nice, I didn't notice that needed an update.  Should we update the home page to also reflect this new page? (change the "read all recent" link? I don't know if it matters

Reply:
To my understanding, this is the entry from the latest page. I'm not sure if it's correct.